### DISCUSSION TITLE
Olá, fico muito feliz em poder contribuir com esse projeto!!

### DISCUSSION THREAD

**Discussion by grmiranda** [DenisPower1/inter#82] [Show and tell]
Estou iniciando junto com alguns colegas um projeto de To-do e optamos por usar o Inter.js pra aprender, segue exemplo:

[todo.html.zip](https://github.com/interjs/inter/files/11901261/todo.html.zip)

**Comment by DenisPower1** [comment]
Muito bom @grmiranda , como foi a experiência? Recomendarias para mais colegas?
